docs/src/test-api/class-testproject.md

```diff
@@ -178,8 +178,9 @@ The directory for each test can be accessed by [`property: TestInfo.snapshotDir`
 
 This path will serve as the base directory for each test file snapshot directory. Setting `snapshotDir` to `'snapshots'`, the [`property: TestInfo.snapshotDir`] would resolve to `snapshots/a.spec.js-snapshots`.
 
-## property: TestProject.snapshotPathTemplate = %%-test-config-snapshot-template-path-%%
+## property: TestProject.snapshotPathTemplate = %%-test-config-snapshot-path-template-%%
 * since: v1.28
+* langs: js
```

Comment:
this and other "langs: js" should not be needed, since the whole file is JS only.